packages/info/__tests__/info.test.ts

```diff
@@ -2,15 +2,15 @@ import info from '../src/index';
 
 describe('info tests', () => {
     it('should log environment info', async () => {
-        const envInfo = await info();
+        const envInfo = await info({});
```

Comment:
Why we need `{}`, can you do default `{}` if nothing passed

Reply:
Okay, will do

Reply:
done